addon/noop-test-waiter.ts

```diff
@@ -0,0 +1,19 @@
+import { IWaiter } from './types';
+
+export default class NoopTestWaiter implements IWaiter {
```

Comment:
Can you create a specific interface that both `TestWaiter` and `NoopTestWaiter` implement? This way TS can verify things for us (that we are properly implementing all the expected methods and whatnot).

Reply:
Done. I like this.